share/spice/quandl/world_bank/quandl_world_bank.js

```diff
@@ -0,0 +1,100 @@
+/**
+* Created with DuckDuckHack.
+* User: brianrisk
+* Date: 2015-02-12
+* Time: 04:37 PM
+* To change this template use Tools | Templates.
+*/
+
+
+(function (env) {
+    'use strict';
+    env.ddg_spice_quandl_world_bank = function(api_result){
+
+        if (!api_result) {
+            return Spice.failed('quandl_world_bank: no results');
+        }
+
+        var result = api_result;
+        
+        // we need two data points to get percent change
+        if (result.data.length < 2) {
+            return Spice.failed('quandl_world_bank: not enough data points');
```

Comment:
same here -- we're expecting the `id` to map it to other internal checks for the given IA